tests/integration/multicluster/remote/main_test.go

```diff
@@ -55,6 +55,7 @@ func TestMain(m *testing.M) {
 		SetupOnEnv(environment.Kube, istio.Setup(&ist, func(cfg *istio.Config) {
 			// Set the control plane values on the config.
 			cfg.ControlPlaneValues = controlPlaneValues
+			cfg.RemoteClusterValues = controlPlaneValues
```

Comment:
this would have defaulted to ControlPlaneValues if you didn't set this right?

Reply:
was wondering what the default behavior should be in this case.  Currently RemoteClusterValues will not default to ControlPlaneValues if not specified

Reply:
What about cases where something should only go to control plane clusters and setting those values for a remote cluster breaks things, but you don't have any values you need to set on the remote clusters. Seems like an edge case but you'd have to set some dummy value for it to be ignored. Explicit seems better to me.

Reply:
My main concern is now every test needs to explicitly declare remote control plane stuff. Don't we want to run all of our tests as multicluster?